File: Android/AutoLuminosity/app/src/main/java/apps/brokenwallsstudios/autoluminosity/model/Room.java
package apps.brokenwallsstudios.autoluminosity.model;

import java.util.ArrayList;

/**
 * Created by brand on 1/8/2016.
 */
public class Room {
    public int Id;
    public int UserId;
    public String Name;
    public String CreateDate;
    public String ExternalId;
    public ArrayList<Light> Lights;
}

File: AutoLuminosityService/AutoLuminosityService/Models/Room.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace AutoLuminosityService.Models
{
    public class Room
    {
        public int Id { get; set; }

        public int UserId { get; set; }

        public string Name { get; set; }

        public string CreateDate { get; set; }

        public string ExternalId { get; set; }

        public List<Light> Lights { get; set; }
    }
}